OpenRA.Mods.Common/Traits/Conditions/GrantConditionOnMovement.cs

```diff
@@ -45,17 +45,59 @@ protected override void Created(Actor self)
 			base.Created(self);
 		}
 
-		void ITick.Tick(Actor self)
+		void INotifyMoving.StartedMoving(Actor self)
```

Comment:
Having three different places that grant/revoke the condition doesn't sit too well with me.  Can we change this to have a single inner grant/revoke method that the other things call?